.github/workflows/data-prepper-log-analytics-basic-grok-e2e-tests.yml

```diff
@@ -13,16 +13,16 @@ jobs:
   build:
     strategy:
       matrix:
-        java: [14]
+        java: [8, 11, 17]
```

Comment:
Why does this e2e also have JDK 8?

Reply:
Data Prepper should support JDK 8. I think the question is, why do the other tests not have JDK 8. :)

Unfortunately there does seem to be a bug with TLS on JDK 8 for gRPC. That appears to be a general issue beyond this PR.

Reply:
I thought since the source code builds on JDK 8 and the tests build on JDK 11, 8 is omitted from other tests. Not a blocker for this PR.